Source/Shaders/PointPrimitiveCollectionFS.glsl

```diff
@@ -0,0 +1,29 @@
+varying vec4 v_color;
+varying vec4 v_outlineColor;
+varying float v_innerPercent;
+varying float v_pixelDistance;
+
+#ifdef RENDER_FOR_PICK
+varying vec4 v_pickColor;
+#endif
+
+void main()
+{
+    float distance = length(gl_PointCoord - vec2(0.5));
```

Comment:
`distance` is the name of a GLSL function.  This code is fine I suppose, but consider a different name.